src/modules/Tab/Tab.js

```diff
@@ -88,31 +102,50 @@ class Tab extends Component {
   }
 
   renderItems() {
-    const { panes, renderActiveOnly } = this.props
+    const { panes, renderActiveOnly, children } = this.props
     const { activeIndex } = this.state
 
-    if (renderActiveOnly) return _.invoke(_.get(panes, `[${activeIndex}]`), 'render', this.props)
-    return _.map(panes, ({ pane }, index) =>
-      TabPane.create(pane, {
+    const isMissingChildren = childrenUtils.isNil(children)
+
+    const items = isMissingChildren ? panes : children
+
+    if (renderActiveOnly) {
+      if (!isMissingChildren) {
+        return _.get(items, `[${activeIndex}]`)
```

Comment:
Note, React children are not always an array.  You would want to use the `React.Children` API for handling children in a more consistent way.